hibernate-core/src/main/java/org/hibernate/Session.java

```diff
@@ -1260,39 +1260,6 @@ public interface Session extends SharedSessionContract, EntityManager {
 	 */
 	<T> NaturalIdMultiLoadAccess<T> byMultipleNaturalId(String entityName);
 
-	/**
-	 * Enable the named {@linkplain Filter filter} for this current session.
-	 * <p>
-	 * The returned {@link Filter} object must be used to bind arguments
-	 * to parameters of the filter, and every parameter must be set before
-	 * any other operation of this session is called.
-	 *
-	 * @param filterName the name of the filter to be enabled.
-	 *
-	 * @return the {@link Filter} instance representing the enabled filter.
-	 *
-	 * @throws UnknownFilterException if there is no such filter
-	 *
-	 * @see org.hibernate.annotations.FilterDef
-	 */
-	Filter enableFilter(String filterName);
```

Comment:
+1
I wasn't sure into which ORM version this will get in and whether it would be ok to break the compatibility or not.
I've added the methods back without the javadocs following an example of 
```
RootGraph<?> createEntityGraph(String graphName);
```